### DISCUSSION TITLE
Contribution activity Issue.

### DISCUSSION THREAD

**Discussion by devconn99** [community/community#36735] [Other Feature Feedback, Questions, & Ideas]
I am using my github account since June 28, 2019.
However, contribution activity is showing from 2016. 

![image](https://user-images.githubusercontent.com/52310638/196850176-2f63386c-d497-466a-b038-37bf8f2d6d28.png)

What's the issue? Is this related to forked repositories? I tried to remove them but the same issue.

How can I hide or remove years(2016-2018) that are no activities?

**Comment by radar-blip** [comment]
I think you may have to contact support to fix that for you.
Here is a link for that: [Support Page](https://support.github.com/request)
Probably under 'Remove data from repository..'

**Reply by devconn99** [reply]
Hi @linuxlawson,

Thanks for your reply. I will contact the support team.

**Reply by radar-blip** [reply]
You're welcome!

**Comment by jfullstackdev** [comment]
I don't see that 

![image](https://user-images.githubusercontent.com/47092464/196995788-e8e3f640-29f7-44c8-a16b-b974e4c11472.png)

**Reply by devconn99** [reply]
It was fixed just ago by the support team.
Thanks for your considering.

**Comment by ossanna16** [comment]
Hey there @devconn99 and thank you for asking a good question! I'm glad you found our community 🙂
